From 75adf581589dcce1a679d63977b1c82d333b0f77 Mon Sep 17 00:00:00 2001
Message-Id: <75adf581589dcce1a679d63977b1c82d333b0f77.1566637431.git.liu.denton@gmail.com>
In-Reply-To: <cover.1566637431.git.liu.denton@gmail.com>
References: <cover.1566637431.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Sat, 24 Aug 2019 04:57:42 -0400
Subject: [PATCH 2/2] completion: add --skip for cherry-pick and revert
To: Git Mailing List <git@vger.kernel.org>

Even though `--skip` is a valid command-line option for cherry-pick and
revert while they are in progress, it is not completed. Add this missing
option to the completion script.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 contrib/completion/git-completion.bash | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/completion/git-completion.bash b/contrib/completion/git-completion.bash
index 62f8ef600d..ebcfb5a5af 100644
--- a/contrib/completion/git-completion.bash
+++ b/contrib/completion/git-completion.bash
@@ -1361,7 +1361,7 @@ _git_checkout ()
 	esac
 }
 
-__git_cherry_pick_revert_inprogress_options="--continue --quit --abort"
+__git_cherry_pick_revert_inprogress_options="--continue --quit --abort --skip"
 
 _git_cherry_pick ()
 {
-- 
2.23.0.248.g3a9dd8fb08

